zcash_client_sqlite/src/chain.rs

```diff
@@ -286,7 +286,7 @@ mod tests {
         validate_combined_chain(db_cache, db_data).unwrap();
 
         // Scan the cache again
-        scan_cached_blocks(db_cache, db_data).unwrap();
+        scan_cached_blocks(db_cache, db_data, None).unwrap();
```

Comment:
can this be  default Parameter so it's not a breaking change?

Reply:
This is the closest I could get to that. Rust doesn't do defaults in the same way swift or kotlin would. The alternative is a separate function but I'm leaning away from that since we're basically the only consumers of this. So it's safer to break things now than it will be next year :)

Reply:
...and I think two functions just adds unnecessary complexity (one function for scanning is simpler)

Reply:
oh, :( I totally agree on breaking things sooner than later